.github/CHANGELOG.md

```diff
@@ -66,6 +66,8 @@ within the defined window. Also, expose some plot parameters and return the figu
 * The function `dm_to_ket` no longer normalizes the density matrix before diagonalizing it.
  [(#180)](https://github.com/XanaduAI/MrMustard/pull/180)
 
+* The internal fock representation of states returns the correct cutoffs in all cases (solves an issue when a pure dm was converted to ket).
```

Comment:
@ziofil, could you elaborate a bit more: what was the issue you found? Did the conversion from pure dm to ket failed to maintain the same cutoffs?

Reply:
yes, the new cutoffs were ignored. e.g. try this:
```python
s = State(ket=SqueezedVacuum(1.0, np.pi / 2).dm([20]))
s.ket([10]).shape
```